python/postprocessing/framework/output.py

```diff
@@ -64,11 +64,35 @@ def write(self):
         self._tree.Write()
 
 class FullOutput(OutputTree):
-    def __init__(self, inputFile, inputTree, outputFile, branchSelection = None, fullClone = False, provenance = False, jsonFilter = None):
+    def __init__(
+            self,
+            inputFile,
+            inputTree,
+            outputFile,
+            branchSelection=None,
+            outputbranchSelection=None,
+            fullClone=False,
+            provenance=False,
+            jsonFilter=None
```

Comment:
any reason for going from one line to multi-line ?

Reply:
very long lines need to be scrolled on github web interface, and are also difficult to read

Reply:
very long lines can be split in fewer lines of reasonable length instead of one line per word... otherwise you now have to scroll vertically instead of having all the code at a glance.

Reply:
for __init_ you see with vertical alignment all arguments at a glance, while a long one-line requires visual parsing (look for comma, look for equal sign, etc).